firebase-vscode/.vscodeignore

```diff
@@ -16,3 +16,4 @@ vsc-extension-quickstart.md
 webpack.*.js
 ../
 *.zip
+node_modules/
```

Comment:
After moving some deps from devdependencies to dependencies in a previous PR, vscode started bundling the dependencies, which are not needed, as webpack should bundle them all.